eng/Version.Details.props

```diff
@@ -5,35 +5,36 @@ This file should be imported by eng/Versions.props
 -->
```

Comment:
please don't merge until a backflow comes with the first build after https://github.com/dotnet/msbuild/pull/12709 is merged